tests/provision/provisioning_test.go

```diff
@@ -0,0 +1,219 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"os"
+	//	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+
+
+
+func TestVps(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Provisioning", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// This test performs VolumePlacementStrategy's replica affinity  of application
+// volume
+var _ = Describe("{ReplicaAffinity}", func() {
+	It("has to schedule app and verify the replication affinity", func() {
+
+		var vpsspec string
+		vpsrules := GetVpsRules()
+
+		for vkey,vrule := range vpsrules {
+			var contexts []*scheduler.Context
+			var volnodes map[string]map[string][]string 
+	
+			var lbldata  [] labelDict 
+			Step("get nodes and set labels: "+vkey, func() {
+				lbldata = getTestLabels(vrule.GetLabels) //TODO: function argument for getting testcase labels
```

Comment:
why do we need this `getTestLabels` wrapper function? All it does it calls the given function. Just call it here directly.